avocado/utils/vmimage.py

```diff
@@ -208,10 +208,7 @@ def __init__(self, version='[0-9]+', build='[0-9]{4}', arch=os.uname()[4]):
         super(CentOSImageProvider, self).__init__(version, build, arch)
         self.url_versions = 'https://cloud.centos.org/centos/'
         self.url_images = self.url_versions + '{version}/images/'
-        if archive.LZMA_CAPABLE:
-            self.image_pattern = 'CentOS-{version}-{arch}-GenericCloud-{build}.qcow2.xz$'
-        else:
-            self.image_pattern = 'CentOS-{version}-{arch}-GenericCloud-{build}.qcow2$'
+        self.image_pattern = 'CentOS-{version}-{arch}-GenericCloud-{build}.qcow2.xz$'
```

Comment:
The commit message would be more appropriate if it stated that "In commit xyz, *conditional* support for lzma was dropped, and it's now always supported".